src/form/Checkbox/CheckboxTheme.ts

```diff
@@ -65,26 +65,60 @@ const baseTheme: Partial<CheckboxTheme> = {
 
 export const checkboxTheme: CheckboxTheme = {
   ...baseTheme,
-  checkbox: [baseTheme.checkbox, 'fill-transparent border border-surface'].join(
-    ' '
-  ),
-  check: [baseTheme.check, 'stroke-primary'].join(' '),
+  checkbox: [
+    baseTheme.checkbox,
+    'fill-transparent border border-surface',
+    'light:group-hover:[&.checked]:fill-blue-600',
+    'light:[&.checked.disabled]:fill-waterloo',
+    'light:group-hover:[&.checked.disabled]:fill-waterloo',
+    'group-hover:[&.checked]:fill-blue-400',
+    'group-hover:[&.checked.disabled]:fill-gray-400',
+    '[&.checked]:fill-blue-500',
+    '[&.checked.disabled]:fill-gray-400 '
+  ].join(' '),
+  check: [
+    baseTheme.check,
+    'light:group-hover:stroke-white',
+    'light:[&.disabled]:stroke-white',
+    'stroke-white',
+    'group-hover:stroke-black',
+    'group-hover:[&.disabled]:stroke-black',
+    '[&.disabled]:stroke-black'
+  ].join(' '),
+  border: [
+    baseTheme.border,
+    'light:stroke-waterloo',
+    'light:group-hover:stroke-blue-600',
+    'light:group-hover:[&.disabled]:stroke-waterloo',
+    'stroke-gray-300',
+    'group-hover:stroke-blue-400',
+    'group-hover:[&.disabled]:stroke-gray-500',
+    '[&.checked]:stroke-blue-500',
+    '[&.disabled]:stroke-gray-500'
+  ].join(' '),
+
   label: {
     ...baseTheme.label,
-    base: [baseTheme.label.base, 'text-surface-content'].join(' ')
+    base: [
+      baseTheme.label.base,
+      'text-surface-content',
+      'light:text-charade',
+      'light:[&.disabled]:text-waterloo',
+      'light:group-hover:[&.disabled]:text-waterloo',
+      'light:[&.checked]:text-vulcan',
+      'light:group-hover:text-blue-400',
+      'dark:text-waterloo',
+      'group-hover:text-blue-300',
+      'group-hover:[&.disabled]:text-waterloo',
+      'group-hover:[&.checked.disabled]:text-athens-gray',
+      '[&.checked]:text-athens-gray'
+    ].join(' ')
   },
   boxVariants: {
```

Comment:
Didn't we just add these props?

Reply:
sorry, I didn't understand the question

Reply:
if you meant move separately "disabled, checked" then it's done